pkg/iam/policy/constants.go

```diff
@@ -22,7 +22,7 @@ import (
 
 // Policy claim constants
 const (
-	PolicyName        = "policy"
+	PolicyName        = "https://min.io/policy"
```

Comment:
not sure why this is changed?

Reply:
I was testing with Auth0 which did not support custom claim with `policy` as the label, i will revert this change.

Reply:
For this, we should bring in support for named policy i.e reserved_prefix + policy name

Reply:
added `policyClaimPrefix` to openid config.